windows-agent/internal/proservices/landscape/executor_test.go

```diff
@@ -312,7 +311,7 @@ func mockRootfsFileServer(t *testing.T, ctx context.Context) string {
 	require.NoError(t, err, "Setup: mockRootfsFileServer could not listen")
 
 	go func() {
-		//nolint:gosec // For the sake of simplicity and readability.
+		//nolint:gosec // ignore G114 for the sake of simplicity and readability.
```

Comment:
ok, so now I see this is for G114. I still think that we can avoid that annotation by adding the timeouts on the server side. Even if this is for a mock, what happens if a test breaks and await forever on the client? Then, we are stuck in our server which may never exists as hanging.

I think something along those lines should be enough:

```
s := &http.Server{
    ReadTimeout: 5 * time.Second,
    WriteTimeout: 5 * time.Second,
}
srv.ListenAndServe()
```